Tests/SwiftParserTest/translated/IfconfigExprTests.swift

```diff
@@ -330,8 +339,17 @@ final class IfconfigExprTests: XCTestCase {
       #endif
       """,
       diagnostics: [
-        DiagnosticSpec(message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion")
-      ]
+        DiagnosticSpec(
+          message: "2nd parameter of canImport should be labeled as _version or _underlyingVersion",
+          fixIts: ["insert '_version'", "insert '_underlyingVersion'", "insert ':'"]
```

Comment:
The way this would presents in an IDE is to offer three different options to fix it:
- `insert '_version'`
- `insert '_underlyingVersion'`
- `insert ':'`

But we only want to offer tow fixes, `insert '_version'` and `insert '_underlyingVersion'` that also insert the `:`. To fix this, instead of creating a separate Fix-It for the colon, you need to add the `.makePresent(node.colon)` change to the other two existing Fix-Its.